manifest.yml

```diff
@@ -4,15 +4,13 @@ applications:
 - name: nyu-inventory-service-sum21
   path: .
   instances: 1
-  memory: 64M
+  memory: 256M
```

Comment:
John mentioned that apps will probably crash with 64MB of RAM, so I bumped it up.